### PR TITLE
first pull request

### PR DESCRIPTION
**Pull request by Aliaksandr-Bychyk**
This is my own installation of environment. In next commit I will improve it by watching the _Udemy courses_.

By now I have successfully created **package.json** file and installed **React**, **Redux**, **React-Redux**, **React-Router**, **Jest**, **React-DOM**. Also I have installed and configurated **Webpack** and **Babel**. For _localhost_ server I have used a **live-server** package. And for the last I have installed and configurated **ESLint**.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,9 +2,37 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "web": "live-server dist",
+    "build": "webpack --mode=development",
+    "build:prod": "webpack --mode=production",
+    "lint": "eslint . --ext .js,.tsx",
+    "lint:fix": "eslint --fix --ext .js,.jsx ."
+  },
+  "dependencies": {
+    "jest": "^26.6.3",
+    "lodash": "^4.17.20",
```
Comment (AliakseiBychyk) [inline package.json]: Do you really need lodash package in your app. Bear in mind it is very heavyweight, and most of the methods you can easily produce with ES6 array methods.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// import _ from 'lodash';
```
Comment (AliakseiBychyk) [inline src/index.js]: Try not to commit commented code. Unless you need it in the future and then comment with `TODO: uncomment code afther ...`

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+//     return element;
+// }
+
+// document.body.appendChild(component());
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const temp = <h1>Hello there!</h1>; // variant to check babel
+// const temp = React.createElement('h1', {className: 'title'}, 'Hello there!'); //right variant
```
Comment (AliakseiBychyk) [inline src/index.js]: In the future create react elements only with JSX

File: package.json (hunk 1 above)
Comment (Aliaksandr-Bychyk) [reply package.json]: Idk, that was in the "getting started" selection of webpack documentation :D 
I thought without Jest it wont work, but it's fine. I think in next commit I'll delete it.

File: src/index.js (hunk 2 above)
Comment (Aliaksandr-Bychyk) [reply src/index.js]: Yep the first eleven lines of code I should have deleted. But 16 and 17 line shows that it works fine with two options of writing code.

File: src/index.js (hunk 3 above)
Comment (Aliaksandr-Bychyk) [reply src/index.js]: ok

**Review by AliakseiBychyk — APPROVED** [review]

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class WebPage extends React.Component {
```
Comment (AliakseiBychyk) [inline src/index.js]: import `Component` using destructuring import

File: package.json [hunk 5]
```diff
@@ -2,9 +2,39 @@
   "name": "reactjs-course-2",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (AliakseiBychyk) [inline package.json]: You should configure test script in that task. Set up `jest` and wright at leas one unit test e.g. for App.js

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```
Comment (AliakseiBychyk) [inline src/index.js]: Rename this file as `App.js`

**Review by AliakseiBychyk — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (AliakseiBychyk) [reply src/index.js]: Do not afraid to rewrite your code or delete some code, you can always see the PR history.

File: src/index.js (hunk 6 above)
Comment (Aliaksandr-Bychyk) [reply src/index.js]: Done!

File: package.json (hunk 5 above)
Comment (Aliaksandr-Bychyk) [reply package.json]: Done!

**Review by AliakseiBychyk — APPROVED** [review]

**Pull request merged**